src/services/node/node.jsx

```diff
@@ -0,0 +1,9 @@
+const EXPENSES_URL = 'https://budget-app-aa.herokuapp.com/expenses'
+
+const getExpenses = () => {
+    return fetch(EXPENSES_URL)
+}
+
+export const serviceNode = {
```

Comment:
a czemu to sie nazywa node, troche nie rozumiem

Reply:
a jak nazwać?